xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/wizard/wizard.css

```diff
@@ -116,7 +116,6 @@
 }
 
 .wizard-body .half{
-  width: 48%;
```

Comment:
Hello. See [this](https://github.com/xwiki/xwiki-platform/pull/436#issuecomment-141450700).